snetd/cmd/channel.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/singnet/snet-daemon/escrow"
+	"github.com/singnet/snet-daemon/etcddb"
+	"github.com/spf13/cobra"
+	"math/big"
+)
+
+// ListChannelsCmd shows list of channels from shared storage
+var ChannelCmd = &cobra.Command{
+	Use:   "channel",
+	Short: "Manage operations on payment channels",
+	Long: "allows us to perform operations on channels with given channel ID." +
+		" User can use 'snetd channel --unlock {channelID}' command to unlock the channel manually.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return RunAndCleanup(cmd, args, newChannelCommand)
+	},
+}
+
+//Channel command type
+type channelCommand struct {
+	etcdclient       *etcddb.EtcdClient
+	paymentChannelId *big.Int
+}
+
+// initializes and returns the new channel command object
+func newChannelCommand(cmd *cobra.Command, args []string, components *Components) (command Command, err error) {
+	channelId, err := getPaymentChannelId(cmd)
+	if err != nil {
+		return
+	}
+	if err != nil {
```

Comment:
The if check is repeating, merge issue maybe

Reply:
removed the duplicate check